sites/game-client/client.ejs

```diff
@@ -248,6 +248,13 @@ absolute URL.
           },
         ));
 
+        const parameters = window.location.search.substring(1).split('&');
+        game.parameters = {};
+        parameters.forEach(parameter => {game.parameters[parameter.split('=')[0]] = parameter.split('=')[1]});
```

Comment:
`parameters` is already available in `query` or `window.query`.
You can try `if (query.test === 'type1')`